lua/entities/gmod_tardis/modules/teleport/sv_tp_fastreturn.lua

```diff
@@ -23,3 +38,10 @@ ENT:AddHook("StopMat", "fastreturn", function(self)
     end
 end)
 
+ENT:AddHook("CanChangeDestination", "fastreturn", function(self, pos, ang)
+    if not self:GetData("fastreturn") then return end
+
+    if self:GetData("teleport") or self:GetData("vortex") then
```

Comment:
This part of the hook should probably be elsewhere

Reply:
the logic here is `fastreturn and (teleport or vortex)`
we cannot split it

Reply:
Ah right, okay!